### ISSUE TITLE
[SIIT] Explore pre-training configurations

### ISSUE THREAD

**Issue by Wintoplay** [OpenThaiGPT/openthaigpt-pretraining#258]
Explore pre-training configurations or other literatures

Example: Optimizer, dataset sampling, and vocab size of tokenizer
tokenizer type: SPM vs BPE
clean data thresold
learning rate (MUP)

![image](https://github.com/OpenThaiGPT/openthaigpt-pretraining/assets/66968226/710958e5-745c-40b0-9b10-be5fbff25bcc)
![image](https://github.com/OpenThaiGPT/openthaigpt-pretraining/assets/66968226/8293616e-820b-49c0-8bf5-f162a22d5522)

Evaluation with real evaluation set

**Comment by Wintoplay** [comment]
[plan.md](https://github.com/OpenThaiGPT/openthaigpt-pretraining/files/12062490/plan.md)
